docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/RegressionSamplesTemplate.ttinclude

```diff
@@ -39,7 +42,7 @@ namespace Samples.Dynamic.Trainers.Regression
             var model = pipeline.Fit(trainingData);
 
             // Create testing data. Use different random seed to make it different from training data.
-            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed:123));
+            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
```

Comment:
>500 [](start = 86, length = 3)

100, maybe?

Reply:
I will do 5 everywhere.

---
In reply to: [275458878](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275458878) [](ancestors = 275458878)